app/src/main/res/values/strings.xml

```diff
@@ -916,53 +916,53 @@
-    <string name="firewall_rule_block_udp_ntp">UDP Blocked</string>
-    <string name="firewall_rule_block_dns_bypass">DNS Bypassed</string>
-    <string name="firewall_rule_exempt_app_bypass_univ">Bypassed App</string>
-    <string name="firewall_rule_exempt_dns_proxied">DNS Proxied</string>
-    <string name="firewall_rule_exempt_orbot_setup">Orbot Setup</string>
+    <string name="firewall_rule_device_lock">Device locked</string>
+    <string name="firewall_rule_foreground">App not in use</string>
+    <string name="firewall_rule_unknown">Source unknown</string>
+    <string name="firewall_rule_block_udp_ntp">UDP blocked</string>
+    <string name="firewall_rule_block_dns_bypass">DNS bypassed</string>
+    <string name="firewall_rule_exempt_app_bypass_univ">Bypassed app</string>
+    <string name="firewall_rule_exempt_dns_proxied">DNS proxied</string>
+    <string name="firewall_rule_exempt_orbot_setup">Orbot set up</string>
     <string name="firewall_rule_block_http">Insecure HTTP</string>
     <string name="firewall_rule_global_lockdown">Lockdown</string>
 
     <string name="firewall_rule_no_rule_desc">No firewall rules matched this connection.</string>
-    <string name="firewall_rule_block_app_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i>.<br/><br/> To change go to <i>All Apps</i> tab.]]></string>
-    <string name="firewall_rule_new_app_desc"><![CDATA[This app made connection for the <i>first</i> time.<br/><br/> To change go to <i>Universal</i> tab.]]></string>
-    <string name="firewall_rule_block_app_unmetered_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> when trying to connect on <i>Unmetered</i> network.<br/><br/> To change go to <i>All Apps</i> tab.]]></string>
-    <string name="firewall_rule_block_app_mobile_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> when trying to connect on <i>Metered</i> network.<br/><br/> To change go to <i>All Apps</i> tab.]]></string>
-    <string name="firewall_rule_block_app_univ_mobile_desc"><![CDATA[A user-set universal firewall rule blocked this <i>app</i> when trying to connect on <i>Metered</i> (Mobile) network.<br/><br/> To change go to <i>Universal</i> tab.]]></string>
-    <string name="firewall_rule_lockdown_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i> connecting to other than <i>bypassed IPs</i>.<br/><br/> To change go to <i>All Apps</i> tab.]]></string>
-    <string name="firewall_rule_block_ip_univ_desc"><![CDATA[A user-set firewall rule blocked the <i>destination IP / Port</i>.<br/><br/> To change go to <i>Universal</i> tab.]]></string>
-    <string name="firewall_rule_block_ip_desc"><![CDATA[A user-set firewall rule blocked the <i>destination IP</i>.<br/><br/> To change go to <i>App-specific Firewall</i> screen.]]></string>
+    <string name="firewall_rule_block_app_desc"><![CDATA[A user-set firewall rule blocked this <i>app</i>.<br/><br/> Go to the <i>All Apps</i> tab to change this.]]></string>
```

Comment:
All these _could_ be said to command the user to do something, as it relies a bit on the context of being the latter sentences.
It is a trade-off to get it to read more naturally, whereas it was perfectly technically sound before, much in the same way all the strings are, but these struck me as too robotic.